app/src/main/res/values/strings.xml

```diff
@@ -474,6 +474,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="undo_confirm_negative">Cancel</string>
     <string name="enable_quest_confirmation_title">Enable this quest type?</string>
     <string name="default_disabled_msg_go_inside">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type.</string>
+    <string name="default_disabled_msg_go_inside_usefulness_warning">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type. Please, do not enable this quest if answer would be "no" for all objects.</string>
```

Comment:
I am not sure if it is StreetComplete's task to warn about "self-imposed" spam. I know, there is the guideline 

> 💤 No spam: It must be possible to determine whether the quest should reasonably be asked. A quest which is to be answered in 99% of the time with the same answer is not a good quest, don't bore the users. 

, however, this concerns the quest design of enabled quests. If the user enables the quest by himself, he is responsible himself for boring himself.

Reply:
It is mostly to discourage someone from spamming mostly useless tag in location where nothing whatsoever is kosher. But that may be overly cautious.

Reply:
I think it is superfluous

Reply:
What about

> This quest type is disabled by default because it is not useful in some regions and you will likely have to enter the place to be able to answer a quest of this type.

?

Reply:
🤷 
Sounds better.